### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by gregorylegarec**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,11 +68,11 @@ $ npm run watch
 
 ### To hack cozy-proxy using the cozy vagrant
 
-- Forward a new port from the virtual machine (for example: `config.vm.network :forwarded_port, guest: 9104, host: 9104` in file Vagrantfile)
-- Go in the shared folder `cd /vagrant` and `cd your-cozy-proxy-folder`
-- `rm -rf node_modules/bcrypt && npm install`
-- Launch cozy-proxy `PORT=9104 HOST="0.0.0.0" npm run watch`
-- You can now access the hacked proxy on `http://localhost:9104` with your navigator
+- Forward cozy-home application port from the virtual machine: `config.vm.network :forwarded_port, guest: 9103, host: 9103` in file Vagrantfile
```
Comment (nono) [inline README.md]: I think you can just say to start the cozy dev vm, with its default configuration

File: README.md [hunk 2]
```diff
@@ -68,11 +68,11 @@ $ npm run watch
...
+- Go to your cozy-proxy folder `cd your-cozy-proxy-folder`
```
Comment (nono) [inline README.md]: You should say explicitely to run this command and those below on your computer, not inside the vm

File: README.md (hunk 1 above)
Comment (gregorylegarec) [inline README.md]: Actually, without adding this error appears in console when running cozy-proxy locally :

````
error - lib:router | Cannot retrieve applications list.
error - lib:router | Cannot retrieve applications list.
error - lib:router | { [Error: connect ECONNREFUSED 127.0.0.1:9103]
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 9103 }
error - lib:router | { [Error: connect ECONNREFUSED 127.0.0.1:9103]
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 9103 }
```

File: README.md (hunk 2 above)
Comment (gregorylegarec) [inline README.md]: Yes, good point.

File: README.md (hunk 1 above)
Comment (nono) [inline README.md]: You are right. I tought it was in the default vagrantfile, but it's not the case

**Pull request merged**